Library/Formula/conoha-iso.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class ConohaIso < Formula
+  desc "Command wrapper to use ISO of ConoHa."
+  homepage "https://github.com/hironobu-s/conoha-iso"
+  url "https://github.com/hironobu-s/conoha-iso/archive/current.tar.gz"
+  version "0.2.1"
+  sha256 "a922d949295ca05f8b3267df471e56bc36590aa9ceacb4ad0d972abdc8192b0a"
+
+  depends_on "go"  => :build
+
+  patch do
+    url "https://gist.githubusercontent.com/neronplex/4770e56b72f363423ed5/raw/6ff8172b4e6627a60b2c353bf752b18b80ad73f3/patch-conoha-iso.diff"
+    sha256 "a5af20f571ee23c3d9ba6c2b604c84ddbd1cc3d7f6f0d1fa43f3722dc6fd9ec3"
+  end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!